portfolio/src/main/webapp/walkthrough/script.js

```diff
@@ -0,0 +1,6 @@
+async function fetchWalkthroughData() {
+    const fetchedDataResponse = await fetch("/data");
+    const fetchedDataText = await fetchedDataResponse.text();
+    const contentElement = document.getElementById("walkthrough-content");
+    contentElement.innerText = fetchedDataText;
+}
```

Comment:
newline 🙀

Reply:
oops.
also no, they don't mention anything about error handling in the walkthrough